libibverbs/man/ibv_wr_post.3.md

```diff
@@ -73,6 +73,9 @@ void ibv_wr_set_sge_list(struct ibv_qp_ex *qp, size_t num_sge,
 void ibv_wr_set_ud_addr(struct ibv_qp_ex *qp, struct ibv_ah *ah,
                         uint32_t remote_qpn, uint32_t remote_qkey);
 void ibv_wr_set_xrc_srqn(struct ibv_qp_ex *qp, uint32_t remote_srqn);
+void ibv_wr_rdma_flush(struct ibv_qp_ex *qp, uint32_t rkey,
+                       uint64_t remote_addr, size_t len,
+                       uint8_t type, uint8_t level);
```

Comment:
I also prefer to use ibv_wr_flush() here.  As Bob mentioned, IBTA SPEC doesn't use the "rdma flush" name.

Reply:
sounds good, updated